frontend/src/services/todoService.ts

```diff
@@ -0,0 +1,58 @@
+import { Category, List } from '../types/common';
+import { v4 as uuidv4 } from 'uuid';
+
+export const todoService = {
+  getAllLists: async (): Promise<Category[]> => {
+    const data = await fetch(`http://localhost:8000/todoList`, {
+      headers: {
```

Comment:
it;s common best practice to have wrapping around your HTTP-request methods. For instance in this case, you can put your header in this wrapper and use only one argument function

Reply:
also, the domain should be in env variables, since the app won't work on any other environments

Reply:
fixed